pallets/multisig/src/lib.rs

```diff
@@ -488,10 +488,9 @@ decl_module! {
             ensure!(Self::is_changing_signers_allowed(&multisig), Error::<T>::ChangeNotAllowed);
             let signers_len: u64 = u64::try_from(signers.len()).unwrap_or_default();
 
-            // NB: the below check can be underflow but that doesn't matter
-            // because the checks in the next loop will fail in that case.
+            let pending_num_of_signers = <NumberOfSigners<T>>::get(&multisig).saturating_sub(signers_len);
```

Comment:
Let's use `checked_sub` here.

Reply:
@maxsam4 I've also added a new error to cover that underflow.